db/mycroft/types/tagging_file_origin_enum.sql

```diff
@@ -0,0 +1,2 @@
+-- How a tagging file is originated.  Used as column type on the tagging.file table
+CREATE TYPE tagging_file_origin_enum AS ENUM ('mycroft', 'selene', 'manual');
```

Comment:
As I have previously mentioned and just an FYI but if you need to add a new enum value this will cause the entire table to have to be reindexed which could take a long time depending on the table size. It is often better to use a separate domain table with a foreign key.